standalone/detectors/fasttrack/include/stacktrace.h

```diff
@@ -1,5 +1,6 @@
-#ifndef _STACKTRACE_H_
-#define _STACKTRACE_H_
+#ifndef STACKTRACE_H
+#define STACKTRACE_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define STACKTRACE_H
```